public/app/features/dashboard-scene/panel-edit/PanelOptions.tsx

```diff
@@ -42,10 +42,10 @@ export const PanelOptions = React.memo<Props>(({ vizManager, searchQuery, listMo
       data,
       plugin: plugin,
       eventBus: panel.getPanelContext().eventBus,
-      instanceState: panel.getPanelContext().instanceState!,
+      instanceState: _pluginInstanceState,
```

Comment:
@mdvictor yea, they do. Only difference is that _pluginInstanceState is part of state (and can be subscribed to). So just figured it was best to use the state that this code is subscribing to here, instead of going via getPanelContext